src/GHCMod.hs

```diff
@@ -277,20 +277,20 @@ File map docs:
 CLI options:
 * `--map-file "file1.hs=file2.hs"` can be used to tell
     ghc-mod that it should take source code for `file1.hs` from `file2.hs`.
-    `file1.hs` can be either full path, or path relative to project root.
+    `file1.hs` can be either full path, or path relative to current directory.
```

Comment:
That's a consistency thing. With cabal projects, there is a project root to speak of. With plain ones, it's a considerably muddier concept, at least as far as I can understand. Personally, I try to set cwd to project root anyway, so for me personally this distinction is not important. Not sure how elisp frontend handles this though.